### ISSUE TITLE
Unable to login

### ISSUE THREAD

**Issue by davecooksey** [instabot-py/instabot.py#1763]
Check this or we will delete your issue. (fill in the checkbox with an X like so: [x])
- [ ] I have searched for other issues with the same problem. (including #1153)

## Hosting
- [ ] Are you hosting the bot on a VPS or other Cloud hosting?
- [ ] If so; have you tried running the code on your local machine?

#### Select one:
- [ ] Bug
- [ ] Feature Request
- [ ] Technical Help
- [ ] Other

### Operating System? (include version)
- [ ] MAC
- [ ] Windows
- [ ] Linux
- [ ] Android

## Python Version?
- [ ] 2.x
- [ ] 3.x

## Description of your issue

```
Include the contents of the error.log here (if applicable). < REPLACE THIS TEXT
```

## Config

```
Include the contents of your config here (REMOVE YOUR USERNAME AND PASSWORD!) (if applicable). < REPLACE THIS TEXT
```

**Comment by iraizo** [comment]
well we need more information

**Comment by AZAH197** [comment]
I think he has the same problem like me. 
I downloaded the ZIP. I updated my userinfo.py.
put my Login data in the example.py file, but when I run the example, its says 
(Date) Login failure
(Date) Connection  error
Removing liked posts

something like that

**Comment by AlanInsam** [comment]
same problem as @AZAH197

**Comment by json144** [comment]
Please help!! Same!!!

**Comment by Brcalvo** [comment]
Yesterday it was working but today when I restarted the bot, it won´t log into Instagram anymore. Maybe they changed the login method?

**Comment by enrifer** [comment]
Traceback (most recent call last):
  File "c:/Users/danie/Desktop/Instagram/instabot.py-master/example.py", line 67, in <module>
    unfollow_whitelist=['example_user_1', 'example_user_2'])
  File "c:\Users\danie\Desktop\Instagram\instabot.py-master\src\instabot.py", line 234, in __init__
    self.login()
  File "c:\Users\danie\Desktop\Instagram\instabot.py-master\src\instabot.py", line 285, in login
    self.s.headers.update({'X-CSRFToken': r.cookies['csrftoken']})
  File "C:\Users\danie\AppData\Local\Programs\Python\Python37-32\lib\site-packages\requests\cookies.py", line 328, in __getitem__
    return self._find_no_duplicates(name)
  File "C:\Users\danie\AppData\Local\Programs\Python\Python37-32\lib\site-packages\requests\cookies.py", line 399, in _find_no_duplicates
    raise KeyError('name=%r, domain=%r, path=%r' % (name, domain, path))
KeyError: "name='csrftoken', domain=None, path=None"

**Comment by BurningNacho** [comment]
> Traceback (most recent call last):
> File "c:/Users/danie/Desktop/Instagram/instabot.py-master/example.py", line 67, in 
> unfollow_whitelist=['example_user_1', 'example_user_2'])
> File "c:\Users\danie\Desktop\Instagram\instabot.py-master\src\instabot.py", line 234, in **init**
> self.login()
> File "c:\Users\danie\Desktop\Instagram\instabot.py-master\src\instabot.py", line 285, in login
> self.s.headers.update({'X-CSRFToken': r.cookies['csrftoken']})
> File "C:\Users\danie\AppData\Local\Programs\Python\Python37-32\lib\site-packages\requests\cookies.py", line 328, in **getitem**
> return self._find_no_duplicates(name)
> File "C:\Users\danie\AppData\Local\Programs\Python\Python37-32\lib\site-packages\requests\cookies.py", line 399, in _find_no_duplicates
> raise KeyError('name=%r, domain=%r, path=%r' % (name, domain, path))
> KeyError: "name='csrftoken', domain=None, path=None"

im having the same problem as you since this morning, i havent found a solution yet :(

**Comment by lenger101** [comment]
Same - Solutions such as #1377 did not solve the issue.

**Comment by mistermagon** [comment]
Same for me, doesn't work and solution #1377 did not solve the issue.

**Comment by Brcalvo** [comment]
Worked for me as well, thanks!

**Comment by lenger101** [comment]
Solved 

> this solved it for me: [#1366 (comment)](https://github.com/instabot-py/instabot.py/issues/1366#issuecomment-400852889)

had issues with editor.

**Comment by DarkSuniuM** [comment]
Closed all the issues till 2019-01-09, We updated the repo, Please create a new issue if you need! Thanks
